Makefile

```diff
@@ -0,0 +1,20 @@
+build-busybox:
+	docker build -t $(TEST_IMAGE):busybox-with-lockfile busybox-with-lockfile
```

Comment:
We should set the default value for `TEST_IMAGE`. Otherwise, this Makefile doesn't work outside of GitHub Actions.

Reply:
I thought we would only use it in GitHub Action, but you are right. I added this in https://github.com/aquasecurity/trivy-test-images/pull/30/commits/0e5f060df03d8db0e1b00533eeafa9e8743b5509.